clog/clog_test.go

```diff
@@ -230,23 +229,22 @@ func TestLogger(t *testing.T) {
 				t.Skip("skipping test; windows os not supported.")
 			}
 
-			tl, err := follower.New(syslogpath, follower.Config{
+			tailLogHandler, err := follower.New(syslogpath, follower.Config{
```

Comment:
Why did you call this a handler?

Reply:
This is kind of a file handler for syslog file (unrelated to goland logging handler). 
Shall rename to tailLogFollower.